test/lightning_web/live/project_live_test.exs

```diff
@@ -1384,7 +1365,7 @@ defmodule LightningWeb.ProjectLiveTest do
                |> element("form#choose_auth_type_form_#{modal_id}")
                |> has_element?()
 
-        credential_name = "#{project_user.role}credentialname"
+        credential_name = Name.generate()
```

Comment:
We should move this into a exmachina sequence..

Reply:
cool. tracked here: https://github.com/OpenFn/Lightning/issues/1713